src/main/java/com/ecommerce/controller/OrderController.java

```diff
@@ -45,10 +45,17 @@ public String showAllOrders(
         return "user/order/index";
     }
 
+    @GetMapping({"/payment", "/new"})
+    public String showPaymentForm(@ModelAttribute("order") @Valid OrderDTO orderDTO, Model model) {
+        orderDTO = orderService.initOrder(orderDTO);
+        model.addAttribute("order", orderDTO);
+        return "user/order/payment";
+    }
+
     @PostMapping
-    public String createNewOrder(@ModelAttribute @Valid OrderDTO orderDTO, BindingResult result, final RedirectAttributes redirectAttributes) {
+    public String createNewOrder(@ModelAttribute("order") @Valid OrderDTO orderDTO, BindingResult result, final RedirectAttributes redirectAttributes) {
```

Comment:
```suggestion
    public String createOrder(@ModelAttribute("order") @Valid OrderDTO orderDTO, BindingResult result, final RedirectAttributes redirectAttributes) {
```
ủa create rồi còn new nữa, nó lạ quá